### DISCUSSION TITLE
[求助]绳网太卡了，有没有好心人给个加速器

### DISCUSSION THREAD

**Discussion by JTLIYOO** [share121/inter-knot#659] [其他]
已经开了steam++每次刷新还是很久，进入后还会出现UI不对、图片加载不出的情况，如图。求个好加速器或者加速器配置。谢谢绳匠们。
![2c277e0d-21f9-41a6-b088-83fd8dea869f](https://github.com/user-attachments/assets/5c8289a3-096e-49f4-a75c-608f8ce61bf3)

**Comment by Sushizhu** [comment]
同，我也是这样的

**Comment by lizard315315** [comment]
+1

**Comment by Yizakl** [comment]
加速选GitHub了吗

**Reply by Sushizhu** [reply]
选了还是这样卡

**Reply by JTLIYOO** [reply]
github的全选了

**Comment by RS64671012** [comment]
+1

**Comment by hzjwork** [comment]
我梯子直接选的原生节点都没用

**Comment by TQSJ-TPR** [comment]
都这样，可能炸了

**Comment by langxmo** [comment]
我的图片一直加载不出，基本是黑的

**Comment by RS64671012** [comment]
@share121

**Comment by JTLIYOO** [comment]
steam++的github加速全勾选了也没有用

**Comment by capacitor1** [comment]
确实卡，但不明白为什么。我的vpn节点换了好几个，就是加载GitHub特别慢，但是去下载国外网盘是能跑满网速的，应该不是我梯子的问题🤔

**Comment by JTLIYOO** [comment]
貌似是绳网的问题，挂着steam++访问GitHub的网页很迅速，但是绳网很卡，等作者更新优化吧

**Comment by share121** [comment]
Github 有带宽限制，只有这么快

**Comment by JTLIYOO** [comment]
图片加载不出看着很难受

**Comment by hjejdjdj** [comment]
更新了现在好多了

**Comment by GanlinSama** [comment]
推荐Clash for windows，也支持安卓端

**Comment by BitterWind** [comment]
推荐使用一元机场..

**Comment by glhongbo** [comment]
我直接裸进的

**Comment by aiTunes-dev** [comment]
可以试试steamcommunity_302，不行的话站长GitHub有插件

**Comment by ilovekillerQueens** [comment]
我是用github翻墙脚本的，蛮流畅的诶，需要吗https://github.com/Alvin9999/new-pac/wiki 
这个是谷歌版坐着还有其他版本

**Comment by HCYYB** [comment]
这个委托我接了！
浏览器搜索“瓦特工具箱”（steam++或者watt toolkit）
根据设备下载
加速“Gihub”
进入绳网并刷新

**Comment by YUI-zzzz** [comment]
这个委托我接了

**Comment by SDCOM-0415** [comment]
这个委托我接了！
我推荐直接用梯子
[地址一](http://piankelaw.top "地址一")
[地址二](http://zheke.top "地址二")
这是**我自己卖**的
#807 #775

**Comment by JiangHuai-H** [comment]
![image](https://github.com/user-attachments/assets/d467e849-baef-4bba-8a29-5f15d858ba79)
这个很好用

**Reply by SDCOM-0415** [reply]
> ![image](https://private-user-images.githubusercontent.com/178008625/356801897-d467e849-baef-4bba-8a29-5f15d858ba79.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyODE1NzcsIm5iZiI6MTcyMzI4MTI3NywicGF0aCI6Ii8xNzgwMDg2MjUvMzU2ODAxODk3LWQ0NjdlODQ5LWJhZWYtNGJiYS04YTI5LTVmMTVkODU4YmE3OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMFQwOTE0MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xODBhOGFlZTViMjlmMWU3MWQ1YmJlZjljNWIwYTBmMjI5ZTNmYzQ2YzY3ZDVjY2VkMTc0MDhlNjU4NjE1M2MwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.jd-BXadNulI7FU5YjWzDfIrtUrZoAYtksuai-q150iQ) 这个很好用

这个就是steam++

**Comment by SDCOM-0415** [comment]
> ![image](https://private-user-images.githubusercontent.com/178008625/356801897-d467e849-baef-4bba-8a29-5f15d858ba79.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMyODE1NzcsIm5iZiI6MTcyMzI4MTI3NywicGF0aCI6Ii8xNzgwMDg2MjUvMzU2ODAxODk3LWQ0NjdlODQ5LWJhZWYtNGJiYS04YTI5LTVmMTVkODU4YmE3OS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxMFQwOTE0MzdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xODBhOGFlZTViMjlmMWU3MWQ1YmJlZjljNWIwYTBmMjI5ZTNmYzQ2YzY3ZDVjY2VkMTc0MDhlNjU4NjE1M2MwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.jd-BXadNulI7FU5YjWzDfIrtUrZoAYtksuai-q150iQ) 这个很好用

这个就是steam++

**Comment by aaa00073** [comment]
可以用雷霆，每天有一个小时的免费时间
